packages/block-editor/src/components/block-pattern-setup/setup-toolbar.js

```diff
@@ -1,7 +1,7 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, isRTL } from '@wordpress/i18n';
```

Comment:
The `SetupToolbar` component is imported by [the `BlockPatternSetup` component](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-pattern-setup/index.js). However, the `BlockPatternSetup` component itself does not seem to be used anywhere.